tools/docker/Dockerfile

```diff
@@ -14,6 +14,7 @@ RUN apt-get -qqy update \
     fluxbox \
     gdebi \
     git \
+    default-jre \
```

Comment:
Why are we adding the JRE here?

Reply:
Part of a different patch to support android emulators.

Reply:
Can you remove it then, if it's a different patch? :)

Reply:
Yes.